operational_analysis/methods/electrical_losses.py

```diff
@@ -158,7 +159,17 @@ def calculate_electrical_losses(self):
         if self._monthly_meter:
             scada_monthly = self._scada_daily.resample('MS')['corrected_energy'].sum().to_frame()
             scada_monthly.columns = ['turbine_energy_kwh']
+
+            # Determine availability for each month represented
+            scada_monthly['count'] = self._scada_sum.resample('MS')['count'].sum()
+            scada_monthly['expected_count_monthly'] = 1 #initialized values for column
+            scada_monthly.loc[((scada_monthly.index.month ==1) | (scada_monthly.index.month == 3) | (scada_monthly.index.month ==5) | (scada_monthly.index.month ==7) | (scada_monthly.index.month ==8) | (scada_monthly.index.month ==10) | (scada_monthly.index.month ==12)), 'expected_count_monthly'] = 44640* self._plant._num_turbines /self._time_conversion[self._plant._scada_freq]
```

Comment:
@nateagarwal Can you walk me through what's happening here? And what is the number 40320 for? Does that hold across different months and different time resolutions of the SCADA data?

Reply:
The lines of code are also too long so you'll need to use the '\' character to put these into multiple lines that fit on the page.

Reply:
@moptis the 40320 represents the number of minutes in a subset of months. By dividing by the scada frequency, I ensure that it accurately represents the number of counts we expect.

I can also happily change the line structure.

Reply:
@moptis I have addressed your changes with line breaks and further comments. 
I also introduced a new case for the leap year correction.
I have some ideas about how to clean this whole up and generalize it, which I intend on submitting in a later PR.